### ISSUE TITLE
Create a dedicated sliceviewer release notes page 

### ISSUE THREAD

**Issue by martyngigg** [mantidproject/mantid#29506]
As a developer we want to announce developments to users. The slicevewer has had many changes over this release cycle and as a result it deserves its own sub page of the workbench release notes.

It should be image heavy to show off all of the new features including:

* non-orthogonal view including peak overlays
* roi/line-cut selection and extraction
* general UX improvements

**Comment by DanielMurphy22** [comment]
There are some additional improvements from others in the team that are currently in mantidworkbench.rst and should be added to this page. Happy to help out where I can for this!

**Comment by DanielMurphy22** [comment]
The release notes from MantidWorkbench.rst that should be added to this new page are:

```
- The Sliceviewer interface has been neatened up to reduce wasted space and devote more to the data view itself.

- Sliceviewer now contains an interactive table of plot details that updates as you move your cursor over the data (functionality from the SpectrumViewer in Mantidplot). This table includes the dimension value and signal for multidimensional workspaces, and for matrix workspaces with a spectrum axis:
   - Signal
   - Spectra Number
   - Detector ID
   - Two Theta
   - Azimuthal Angle
   - Time of Flight
   - Wavelength
   - Energy
   - d-Spacing
   - Mod Q

- On opening, the zoom tool is selected by default.
- Reversed colourmaps are now accessed with a reverse checkbox.
- The default colourmap can be set in File > Settings.
- The scale (lin/log) remains set when reopened.

- Sub-plots in the sliceviewer now follow the scaling on the colorbar.
- When line plots are active on the sliceviewer the arrow keys can now be used to move the cursor a pixel at a time.
- The line width of the line plots has been reduced.

- Figure options have been removed as most options were not appropriate.
```

**Comment by DanielMurphy22** [comment]
For my own reference:

- SliceViewer can now correctly display non-orthogonal axes for output of the MDNorm algorithm.

**Comment by DavidFair** [comment]
Closed by #29591
